wordcount/src/wordcount/Main.java

```diff
@@ -0,0 +1,54 @@
+
+        String[] words = unText.replaceAll("\\p{Punct}","").toLowerCase().split("\\s+");
+
+        //Create a HashMap: 
+        System.out.println("*** HashMaps ***");
+		HashMap<String, Integer> hashMap = new HashMap<String, Integer>();
+
+		//For Loop
+		for ( String s : words)
+		{
+            int hashcount = hashMap.containsKey(s) ? hashMap.get(s) : 0;
+            hashMap.put(s, hashcount +1);
+
+
+		    // System.out.println(s);
+
+		}
```

Comment:
I like how you are getting in the habit of commenting your code. Maybe try to be a bit more descriptive.For example: What is this for loop doing?